GCEWindowsAgent/wsfc_test.go

```diff
@@ -54,16 +56,14 @@ func TestNewWsfcManager(t *testing.T) {
 		want *wsfcManager
 	}{
 		{"empty meta config", args{&testMetadata}, &wsfcManager{agentNewState: stopped, agentNewPort: wsfcDefaultAgentPort, agent: testAgent}},
-		{"wsfc enabled", args{setEnableWSFC(testMetadata, true)}, &wsfcManager{agentNewState: running, agentNewPort: wsfcDefaultAgentPort, agent: testAgent}},
+		{"wsfc enabled", args{setEnableWSFC(testMetadata, "true")}, &wsfcManager{agentNewState: running, agentNewPort: wsfcDefaultAgentPort, agent: testAgent}},
 		{"wsfc addrs is set", args{setWSFCAddresses(testMetadata, "0.0.0.0")}, &wsfcManager{agentNewState: running, agentNewPort: wsfcDefaultAgentPort, agent: testAgent}},
 		{"wsfc port is set", args{setWSFCAgentPort(testMetadata, "1818")}, &wsfcManager{agentNewState: stopped, agentNewPort: "1818", agent: testAgent}},
 	}
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			if got := newWsfcManager(tt.args.newMetadata); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("newWsfcManager() = %v, want %v", got, tt.want)
-			}
-		})
+		if got := newWsfcManager(tt.args.newMetadata, ini.Empty()); !reflect.DeepEqual(got, tt.want) {
```

Comment:
In general I'm not looking closely at the tests. That said, unless `got` means Game of Thrones, I think you should rename it to something else (found? v?).